lms/startup.py

```diff
@@ -58,7 +58,7 @@ def add_mimetypes():
     mimetypes.add_type('application/font-woff', '.woff')
 
 
-def enable_theme():
+def enable_stanford_theme():
```

Comment:
The plan is to deprecate and remove Stanford theming in the near future, but there has to be some overlap of supporting both Stanford theming and this new comprehensive theming system, so that people who are currently using Stanford theming have some time to migrate over to the new system. Believe me, I would love to rip out Stanford theming as soon as possible, but we can't just yet.

It's better to call it `enable_stanford_theme` because everyone refers to it as "Stanford theming", and there are already multiple references in the codebase to "Stanford theming", as well. When we're confident that the new theming system can do everything that Stanford theming can do, then we'll declare Stanford theming deprecated, help people migrate to the new theming system, and start removing Stanford theming from the codebase.